src/sdk/namespace_nb.js

```diff
@@ -58,9 +58,18 @@ class NamespaceNB {
     // OBJECT LIST //
     /////////////////
 
-    list_objects(params, object_sdk) {
+    async list_objects(params, object_sdk) {
         if (this.target_bucket) params = _.defaults({ bucket: this.target_bucket }, params);
-        return object_sdk.rpc_client.object.list_objects(params);
+        const object_info = await object_sdk.rpc_client.object.list_objects(params);
+        let bucket_info;
+        try {
+            bucket_info = await object_sdk.read_bucket_sdk_config_info(params.bucket);
+        } catch (err) {
+            if (err.rpc_code !== 'NO_SUCH_BUCKET') throw err;
+        }
+        object_info.object_owner_name = this.get_object_owner_name(bucket_info.bucket_info.name.unwrap());
+        object_info.object_owner_id = bucket_info._id;
```

Comment:
As we discussed, the owner should be the bucket owner account name and not the bucket name itself.

Reply:
Sure. will return the obect owner name directly coming from the list objects